bolero/representation/test/test_ul_policies.py

```diff
@@ -30,6 +31,44 @@ def test_linear_gaussian():
     assert_almost_equal(np.std(Y_sampled), 1.0, places=1)
 
 
+def test_constant_gaussian_full_covariance():
+    random_state = np.random.RandomState(0)
+
+    n_samples = 10000
+    n_weights = 5
+    mean = np.ones(n_weights)
+    ulp = ConstantGaussianPolicy(
+        n_weights, covariance="full", mean=mean,
+        covariance_scale=1.0, random_state=random_state)
+    Y = mean + random_state.randn(n_samples, n_weights)
+    ulp.fit(None, Y, np.ones(n_samples))
+    estimated_mean = ulp(explore=False)
+    assert_array_almost_equal(mean, estimated_mean, decimal=2)
+
+    p = ulp.probabilities([mean])
+    p2 = ulp.probabilities([mean + 1.0])
```

Comment:
same here, variables p and p2 should have names indicating what probabilities they store